src/Helpers/Configuration.ts

```diff
@@ -0,0 +1,6 @@
+import 'Extensions/Array'
+
+export const keysToScreamingSnakeCase = (object: object = {}): {} =>
+  Object.entries(object)
+    .map(([key]) => [key.replace(/([A-Z])/g, capital => '_' + capital).toUpperCase(), key])
+    .toObject()
```

Comment:
After we get ramda package added to the node in #317, you can use [fromPairs](https://ramdajs.com/docs/#fromPairs) instead of using the custom Array method `.toObject`